ansible/roles/invoker/tasks/deploy.yml

```diff
@@ -290,6 +290,22 @@
   set_fact:
     env: "{{ env | combine(invoker.extraEnv) }}"
 
+- name: setup elasticsearch activation store env
+  set_fact:
+    elastic_env:
+      "CONFIG_whisk_activationStore_elasticsearch_protocol": "{{ db.elasticsearch.protocol}}"
```

Comment:
Better to document the config in `application.conf` with some explanation for various config options as done for other stores. This would enable users (not using ansible) to configure this store easily

https://github.com/apache/openwhisk/blob/8eb922f568d7843cdb1068252d29ba5723b30283/common/scala/src/main/resources/application.conf#L216-L230

Reply:
great, I missed this